### DISCUSSION TITLE
Installation error TameMyCerts.dll not found

### DISCUSSION THREAD

**Discussion by lde0680** [Sleepw4lker/TameMyCerts#28] [General]
Hello, I'm trying to install the module but have an error message about missing TameMyCerts.dll in source directory.
I'm unable to find it on github repositoryn where can we find it?

Error 👍 
PS C:\\TameMyCerts-source\TameMyCerts> .\Install.ps1 -PolicyDirectory C:\TameMycert-PolicyFiles
C:\TameMyCerts-source\TameMyCerts\install.ps1 : Could not find
C:\TameMyCerts-source\TameMyCerts\TameMyCerts.dll
At line:1 char:1
+ .\Install.ps1 -PolicyDirectory C:\TameMycert-PolicyFiles
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,install.ps1

Thanks for your help

**Comment by lde0680** [comment]
Hello, sorry I have downloaded install package and it is now OK

**Comment by lde0680** [comment]
solved

**Comment by ldc-2024** [comment]
Sorry, i have the same issue. im not able to find the install package. Where is it?

**Comment by Sleepw4lker** [comment]
here: https://github.com/Sleepw4lker/TameMyCerts/releases

**Reply by ldc-2024** [reply]
thank you!
